pom.xml

```diff
@@ -391,6 +507,36 @@
         <scope>test</scope>
       </dependency>
 
+      <dependency>
+        <groupId>org.jglue.cdi-unit</groupId>
+        <artifactId>cdi-unit</artifactId>
+        <version>3.1.2</version>
+        <scope>test</scope>
+        <exclusions>
+          <exclusion>
```

Comment:
It might not be needed, but I'm just going by http://jglue.org/cdi-unit-user-guide/#weldVersions